articles/covid-tracker-in-vue.md

```diff
@@ -0,0 +1,1100 @@
+  differently to styling HTML elements.
+
+### Grouping SVG elements
+
+In the example above, the rectangle and text elements are direct children of the
+SVG element—but what if we have multiple rectangles each with some text
+associated with them? We could have a `rect` followed by a `text` followed by a
+`rect`, by a `text`, etc, but that's not especially nice—especially when we want
+to start looping through data in Vue.
+
+SVG has an element for grouping other elements, `<g>`. It's kind of similar to
+HTML's `<div>` element. Here's how we use it:
+
+```html
+<svg
+  width="400" height="250"
+  xmlns="http://www.w3.org/2000/svg">
+  <g transform="translate(50, 50)">
+    <rect
+      width="200" height="100"
+      fill="hsl(10, 80%, 70%)"
+    />
+    <text>Some text</text>
+  </g>
+</svg>
+```
+
+This outputs exactly the same as the example above that used `x` and `y`, but
+without the duplication of the positions as the `transform` is also being
+applied to the elements within the group. Note that the group element doesn't
+accept `x` and `y` attributes: you have to use a transform to translate the
+group instead.
+
+Now that we've freed up the `x` and `y` attributes of the text, let's use it to
+position the text in the centre of the rectangle:
+
+```html
+<svg
+  width="400" height="250"
+  xmlns="http://www.w3.org/2000/svg">
+  <g transform="translate(50, 50)">
+    <rect
+      width="200" height="100"
+      fill="hsl(10, 80%, 70%)"
+    />
+    <text
+      x="100" y="50"
+      alignment-baseline="middle"
+      text-anchor="middle">
+      Some text
+    </text>
+  </g>
+</svg>
+```
+
+Here's the output:
+
+<svg width="400" height="250" xmlns="http://www.w3.org/2000/svg" style="border: 1px rgba(41, 61, 163, 0.2) solid;">
+<g transform="translate(50, 50)">
+  <rect width="200" height="100" fill="hsl(10, 80%, 70%)" />
+  <text x="100" y="50" alignment-baseline="middle" text-anchor="middle">
+    Some text
+  </text>
+</g>
+</svg>
+
+There's a couple things going on here, including two new attributes. First, note
+that the x and y positions of the text are set to half the width and height of
+the rectangle. That means that the text will be positioned from the centre of
+the rectangle. By default, though, that would result in the text appearing in
+the upper right of the rectangle, like this:
+
+<svg
+  width="400" height="250"
+  xmlns="http://www.w3.org/2000/svg" style="border: 1px rgba(41, 61, 163, 0.2) solid;">
+  <g transform="translate(50, 50)">
+    <rect width="200" height="100" fill="hsl(10, 80%, 70%)" />
+    <text x="100" y="50">Some text</text>
+  </g>
+</svg>
+
+The two attributes added in the last example help us here. `alignment-baseline`
+says where the text should be positioned vertically by specifying which baseline
+of the element should be used. `text-anchor` does a similar thing, but
+horizontally instead of vertically.
+
+Here's a quick demo:
+
+@todo WHY ISN'T ALIGNMENT BASELINE WORKING!
```

Comment:
Some rendering thing? Have you tried nextTick()?

Reply:
it looks like a browser bug - it doesn't work at all in firefox 🤔

Reply:
fixed by changing to dominant-baseline